src/client/post/Write/editorActions.js

```diff
@@ -165,6 +166,8 @@ export function createPost(postData) {
 
     const newBody = isUpdating ? getBodyPatchIfSmaller(postData.originalBody, body) : body;
 
+    saveSettings({ upvoteSetting: upvote, rewardSetting: reward });
```

Comment:
This action isn't dispatched so it won't save anything, am I right?

Reply:
yup you're right... it needs to be `dispatch(saveSettings({ upvoteSetting: upvote, rewardSetting: reward })`